pilot/pkg/model/push_context_test.go

```diff
@@ -1205,8 +1205,8 @@ func TestWasmPlugins(t *testing.T) {
 			expectedExtensions: map[extensions.PluginPhase][]*WasmPluginWrapper{
 				extensions.PluginPhase_AUTHN: {
 					convertToWasmPluginWrapper(wasmPlugins["authn-med-prio-all"]),
-					convertToWasmPluginWrapper(wasmPlugins["authn-low-prio-all-network"]),
 					convertToWasmPluginWrapper(wasmPlugins["authn-low-prio-all"]),
+					convertToWasmPluginWrapper(wasmPlugins["authn-low-prio-all-network"]),
```

Comment:
Not sure if this should be defined as a behavior change.

i: authn-low-prio-all/testns-1
j: authn-low-prio-all-network/testns-1

old behavior:
compare `{name}/{namespace}`, i < j got `false`.

new behavior:
compare name first, then namespace. Since name i is less than name j,  i < j got `true`.